### DISCUSSION TITLE
Cannot open Flows editor

### DISCUSSION THREAD

**Discussion by PixelTaro** [illacloud/illa-builder#3698] [English Support]
From illacloud dashboard, I go to Flows and tried to create a new Flow. The browser open the editor in a new tab but it keeps loading status and nothing appear. I tried to clear cache and different browsers but the issue is the same. However, on the Flows dashboard, a new Flow appears but I cannot do anything with it.

Does anyone know what happens or how to get over it?

![Screenshot 2024-02-16 at 16 09 32](https://github.com/illacloud/illa-builder/assets/102939411/2bbf2c03-83ce-4f6c-804e-512ae53487bc)

**Comment by AruSeito** [comment]
Hello,plz try again
